common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -293,49 +300,53 @@ div.video {
           font-size: em(14);
         }
 
-        li {
+        a {
```

Comment:
Where possible, using classes instead of element styles is preferred. If we can use an existing class (or add one) that would be preferred to applying this to all "a" elements at this level.